jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/LogEventHandler.java

```diff
@@ -17,40 +17,53 @@
 package com.google.cloud.tools.jib.maven;
 
 import com.google.cloud.tools.jib.event.events.LogEvent;
+import com.google.cloud.tools.jib.plugins.common.AnsiLoggerWithFooter;
 import java.util.function.Consumer;
 import org.apache.maven.plugin.logging.Log;
 
 /** Handles {@link LogEvent}s by passing to the Maven {@link Log}. */
 class LogEventHandler implements Consumer<LogEvent> {
 
   private final Log log;
+  private final AnsiLoggerWithFooter ansiLoggerWithFooter;
 
-  LogEventHandler(Log log) {
+  LogEventHandler(Log log, AnsiLoggerWithFooter ansiLoggerWithFooter) {
     this.log = log;
+    this.ansiLoggerWithFooter = ansiLoggerWithFooter;
   }
 
   @Override
   public void accept(LogEvent logEvent) {
     switch (logEvent.getLevel()) {
       case LIFECYCLE:
-        log.info(logEvent.getMessage());
+        if (log.isInfoEnabled()) {
+          ansiLoggerWithFooter.log(log::info, logEvent.getMessage());
+        }
         break;
 
       case DEBUG:
-        log.debug(logEvent.getMessage());
+        if (log.isDebugEnabled()) {
+          ansiLoggerWithFooter.log(log::debug, logEvent.getMessage());
+        }
         break;
 
       case ERROR:
-        log.error(logEvent.getMessage());
+        if (log.isErrorEnabled()) {
+          ansiLoggerWithFooter.log(log::error, logEvent.getMessage());
+        }
         break;
 
       case INFO:
         // Use lifecycle for progress-indicating messages.
-        log.debug(logEvent.getMessage());
+        if (log.isDebugEnabled()) {
```

Comment:
Huh... interesting comment that we send INFO to debug. Should it really be the case?

Reply:
Yeah, the `INFO` here is at a higher verbosity than `LIFECYCLE`, and in Maven, the lowest verbosity is `.info`, so the next verbosity would be `.debug`.